libhb/handbrake/common.h

```diff
@@ -703,6 +703,8 @@ struct hb_job_s
     {
         int decode;
         int async_depth;
+        int filters_count;
```

Comment:
do we really need this variable?

Reply:
we need to save number of filters after sanitize_filter_list call. 
hb_qsv_full_path_is_enabled function is called several times.

Reply:
Done!